agent/src/agent/pipeline/schema.py

```diff
@@ -47,6 +51,12 @@ def create(schema: dict) -> dict:
     return AnodotApiClient(destination.repository.get()).create_schema(schema)['schema']
 
 
+def update(schema: dict) -> dict:
+    # deletes schema and recreates with the same id field
+    delete(schema['id'])
```

Comment:
I don't understand how it works, you delete schema id here but then you get `schema['id']` in the `update_schema` method

Reply:
Here `delete(schema['id'])` deletes schema from Anodot by its 'id'
And after that creates new schema with the same 'id'

Reply:
ahaaa, right, understood :)